Exiled.API/Features/Roles/Scp049Role.cs

```diff
@@ -96,6 +114,16 @@ internal Scp049Role(Scp049GameRole baseRole)
         /// </summary>
         public Ragdoll RecallingRagdoll => Ragdoll.Get(ResurrectAbility.CurRagdoll);
 
+        /// <summary>
+        /// Gets all the dead zombies.
+        /// </summary>
+        public HashSet<uint> DeadZombies => Scp049ResurrectAbility.DeadZombies;
```

Comment:
We should make the minimal effort required to convert these two dictionaries to use a Player object instead of the PlayerID